packages/lib-user/src/components/MyGroups/MyGroupsContainer.js

```diff
@@ -71,21 +70,23 @@ function MyGroupsContainer({
         }
       >
         <ContentBox
-          linkLabel='Learn more about Groups'
-          linkProps={{ href: '/groups' }}
```

Comment:
Removed because there is nothing at https://fe-root.preview.zooniverse.org/groups